serbian/burekSir/burekSir.md

```diff
@@ -1,8 +1,10 @@
 "Da'l je moguće? Ne mogu da verujem. Kako nemate? Neću sir!" - razvijao se bes iz njegovih očiju.
 Galamom privukao je pažnju momaka koji su već tu sedeli i zadovoljno jeli svoje bureke s mesom i pili jogurt "Granice".
 
-Konacno, nakon sto je uzeo burek sasirom, nezadovoljno je poceo da ga jede. Sira nije bilo!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.